.env.development

```diff
@@ -8,12 +8,12 @@ REACT_APP_IMAGE_SIZE=20
 REACT_APP_LOG_LEVEL=debug
 REACT_APP_KEYCLOACK_ADMIN=http://keycloak:8080/admin/realms/default
 REACT_APP_ORION=http://localhost:1026
-
+REACT_APP_GOOGLE_MAPS=INSERTKEYHERE
 
 MONGO_DB=mongodb://localhost:27017/graphql
 CONFIGURATION_API_PORT=4000
 LOGOPS_LEVEL=DEBUG
-JWKS_URL=http://keycloak:8080/realms/default/protocol/openid-connect/certs
```

Comment:
are you sure this is a good idea? won't this impact the token you get and then how this is recognized by the keycloak interaction during tenant creation?

Reply:
https://github.com/orchestracities/auth-management-ui#development-set-up

> NOTE: Tenant creation won't work due to the issuer different between docker (keycloak) and your local environment (localhost), you can solve this by adding and entry in your /etc/hosts file that points 127.0.0.1 to keycloak and using using keycloak as hostname in REACT_APP_KEYCLOACK_ADMIN and REACT_APP_OIDC_ISSUER. This is already the default value in case of .env.docker.

Reply:
probably this affects only the interaction of configuration api, one more reason anymore to be consistent and keep `keycloak`